AdminUi/src/AdminUi/ClientApp/src/app/services/identity-service/identity.service.ts

```diff
@@ -36,6 +36,19 @@ export interface Identity {
     createdAt: Date;
     identityVersion: string;
     quotas: Quota[];
+    devices: Device[];
+}
+
+export interface Device {
+    id: string;
+    username: string;
+    createdAt: Date;
+    lastLogin: LastLoginInformation;
+    createdByDevice: string;
+}
+
+export interface LastLoginInformation {
+    time: Date;
```

Comment:
In the Backend, this property is nullable. So it should be nullable here as well.